lib/wasi/src/syscalls/wasix/futex_wake_all.rs

```diff
@@ -6,7 +6,7 @@ use crate::syscalls::*;
 /// ## Parameters
 ///
 /// * `futex` - Memory location that holds a futex that others may be waiting on
-#[instrument(level = "trace", skip_all, fields(futex_idx = field::Empty, woken = field::Empty), ret)]
+//#[instrument(level = "trace", skip_all, fields(futex_idx = field::Empty, woken = field::Empty), ret)]
```

Comment:
No more tracing here? Perhaps better to remove?

Reply:
that's a mistake, ill undo that

Reply:
fixed